backend/migrations/versions/af0437dab922__add_dataallManaged_flag.py

```diff
@@ -0,0 +1,24 @@
+"""_add_dataallManaged_flag
+
+Revision ID: af0437dab922
+Revises: f6cd4ba7dd8d
+Create Date: 2024-02-15 10:42:06.833990
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'af0437dab922'
+down_revision = 'f6cd4ba7dd8d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('consumptionrole', sa.Column('dataallManaged', sa.Boolean(), nullable=False))
```

Comment:
How are we going to handle pre-existing consumption roles? Maybe we need to backfill the values of this column or set a default = `True` for existing roles. I think with the condition `nullable=False` we are going to run into RDS consistency problems otherwise